model/src/main/java/com/basistech/rosette/apimodel/Address.java

```diff
@@ -117,14 +115,4 @@
      * @return the address P.O. Box
      */
     private final String poBox;
-
-    /**
-     * @return the extra properties of the address
-     */
-    private final Map<String, String> extra;
-
-    /**
-     * @return the address uid
-     */
-    private final String uid;
```

Comment:
just to make sure this is intentional. @ChiniSinha

Reply:
@Katsuya-Tomioka yes, we decided to omit these since they don't actually contribute to the match score (and uids are also not present for `Name` objects)